CHANGES_NEXT_RELEASE

```diff
@@ -2,3 +2,4 @@
 - Fix: queue workers deadlock when MQTT broker was not responding to connection attemp
 - Fix: Ngsiv2-AttrsFormat header set to "normalized" in NGSI patching custom notification
 - Fix: Orion should return error and refuse to start when port is negative (3875)
+- Fix: logLineMaxSize and logLineMaxSize in the log admin REST API (3707)
```

Comment:
Text-style:

```suggestion'
- Fix: Orion should return error and refuse to start when port is negative (#3875)
- Fix: logLineMaxSize and logLineMaxSize in the log admin REST API (#3707)
```

Reply:
Fix in [18cb428](https://github.com/telefonicaid/fiware-orion/pull/4289/commits/18cb428517740d4662bb8f7a325f4ca6e7692b8a)